lib/cc/analyzer/path_patterns.rb

```diff
@@ -10,30 +10,40 @@ def expanded
         @expanded ||= expand
       end
 
+      def match?(path)
+        @patterns.any? do |pattern|
+          value = glob_value(pattern)
+          File.fnmatch(pattern, path)
```

Comment:
Sorry, I'm not sure if that answers my concerns.

This implementation will require `pattern` to be (.e.g) `**.rb` (not `**/*.rb`) to match a top-level files like `foo.rb` -- is that the behavior you want / need? It's usually not, but it may be; just want to be sure.